code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2268,3 +2268,20 @@ All effects don't start immediately, but rather get worse over time; the rate is
 /datum/reagent/consumable/ethanol/pruno/on_mob_life(mob/living/carbon/M)
 	M.adjust_disgust(5)
 	..()
+
+/datum/reagent/consumable/ethanol/e_tank
+	name = "E Tank"
+	description = "A weirdly named energy drink. Tastes tropical and refills your energy!"
+	boozepwr = 90 //robots may not feel it but this is STRONG
+	color = "#5555FF"
+	quality = DRINK_FANTASTIC //atleast its a good drink
+	taste_description = "tropical energy"
+	glass_icon_state = "etank"
+	glass_name = "E Tank"
+	glass_desc = "A weirdly named energy drink can. Tastes tropical and refills your energy!"
+	metabolization_rate = 2.5 * REAGENTS_METABOLISM //goes away quickly, 1u for 2 stamina heals.
+
+/datum/reagent/consumable/ethanol/e_tank/on_mob_life(mob/living/carbon/C)
+	C.adjustStaminaLoss(-2*REM, 0)
+	to_chat(C, "<span class='danger'>You feel your energy restored!</span>")
```

Comment:
This is going to be spamming chat every cycle, I think. You should probably just remove it.

Reply:
that is... true

Im sorry little one.

Reply:
You could just add a chance for it